tbx/project_styleguide/templates/patterns/navigation/components/includes/subnav-child-menu.html

```diff
@@ -2,13 +2,23 @@
 
 <div class="sub-nav-desktop sub-nav-desktop--child">
     <ul class="sub-nav-desktop__list sub-nav-desktop__list--child">
-        {% for page in pages %}
+        {% for gchild in child.value.child_links %}
             <li class="sub-nav-desktop__item">
-                <a class="sub-nav-desktop__link sub-nav-desktop__link--child sub-nav-desktop__link--no-children" href="{% pageurl page %}" {% if last_menu_item and forloop.last %}data-last-menu-item-desktop{% endif %}>
-                    {{ page.nav_text }}
+                <a href="{{ gchild.value.url }}" class="sub-nav-desktop__link">
+                    <span class="sub-nav-desktop__link-text">{{ gchild.value.text }}</span><span class="link-columns__tail">&nbsp;{% include "patterns/atoms/icons/icon.html" with name="arrow" classname="sub-nav-desktop__arrow" %}</span>
```

Comment:
Suggestion: I think the `link-columns__tail` class will either need renaming or the `link-columns` class added on a parent element for valid BEM.

Reply:
forgot to rename that one - thanks